l10n/en/firefox/installer-help.ftl

```diff
@@ -0,0 +1,31 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/installer-help/
+
+## Required
+installer-help-page-title = Your download was interrupted
+installer-help-main-heading = Hmm… your download was interrupted
+installer-help-main-tagline = That happens sometimes. Let’s try something different.
+installer-help-action-channel = This download should work better for you.
+installer-help-action-choose = Choose a download below.
+
+##
+installer-help-firefox-release-title = { -brand-name-firefox-browser }
+
+# Variables:
+#   $trackers (number) - number of trackers blocked by Firefox
+installer-help-firefox-release-desc = Get the latest. Automatic privacy is here. Download { -brand-name-firefox } to block over { $trackers } trackers.
```

Comment:
I see this is using `-brand-name-firefox` and above uses `-brand-name-firefox-browser`. What's the difference?

Reply:
Hmm yeah - this does seem inconsistent. Let me update to use Firefox Browser for completeness.